src/frontend/src/lib/types/token.ts

```diff
@@ -27,3 +27,9 @@ export type OptionTokenId = TokenId | undefined | null;
 export type OptionTokenStandard = TokenStandard | undefined | null;
 
 export type TokenToPin = Pick<Token, 'id'> & { network: Pick<Token['network'], 'id'> };
+
+interface TokenFinancialData {
+	usdValue: number | undefined;
+}
+
+export type TokenWithFinancials = Token & TokenFinancialData;
```

Comment:
I like `TokenUi`, it allows more space to add things. About `balance`, yes! My first iteration had that, but I thought to separate the PRs, even because if i put balance there, I need to modify more usage and the PR would have a lot more files changed. WDYT?

Reply:
Doing it in PR #1798